src/plugins/dashboard/public/application/embeddable/viewport/dashboard_viewport.tsx

```diff
@@ -162,13 +132,8 @@ export class DashboardViewport extends React.Component<DashboardViewportProps, S
                 isReadonlyMode={
                   this.props.container.getInput().dashboardCapabilities?.hideWriteControls
                 }
-                onLinkClick={
-                  isEditMode
-                    ? this.addFromLibrary
-                    : () => this.props.switchViewMode?.(ViewMode.EDIT)
-                }
-                showLinkToVisualize={isEditMode}
-                onVisualizeClick={this.createNewEmbeddable}
+                onLinkClick={() => this.props.switchViewMode?.(ViewMode.EDIT)}
```

Comment:
I'm glad this PR removes the 'Create New' button from the empty screen - and the new design is looking great! If at all possible, maybe in a followup, it would also be great to remove the 'switchViewMode` behaviour. This method is injected in a non-standard way and is the only place where the dashboard state manager is coupled with the dashboard container.